upgrade/v2.5/script/upgrade.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
+
+# Set the ENV var for KDD backend
+export DATASTORE_TYPE=kubernetes
+
+# Create the CRDs by applying the crds.yaml manifest.
+/sbin/kubectl apply -f crds.yaml
+
+# Get IPPools list with old calicoctl (from TPR) and save it as a yaml file.
+# Apply the yaml file with the new calicoctl (to CRD)
+/sbin/calicoctl-v1.4 get ippool -o yaml > ippool.yaml
+/sbin/calicoctl-v1.5 apply -f ippool.yaml
+
+# Get BGPPeers list with old calicoctl (from TPR) and save it as a yaml file.
+# Apply the yaml file with the new calicoctl (to CRD)
+/sbin/calicoctl-v1.4 get globalbgppeer -o yaml > bgppeer.yaml
+/sbin/calicoctl-v1.5 apply -f bgppeer.yaml
+
+# List all the Felix configs using kubectl and save it in a yaml file.
+# Change the apiVersion from 'projectcalico.org/v1' to 'crd.projectcalico.org/v1' and save it in a new yaml file.
+# Apply the modified yaml file.
+/sbin/kubectl get globalconfig --all-namespaces -o yaml > cat tpr-felixconfig.yaml
+cat tpr-felixconfig.yaml | sed '/apiVersion/s/projectcalico\.org\/v1/crd\.projectcalico\.org\/v1/g' | sed '/kind/s/GlobalConfig/GlobalFelixConfig/g' > crd-felixconfig.yaml
+/sbin/kubectl apply -f crd-felixconfig.yaml
+
+# List all the BGP configs using kubectl and save it in a yaml file.
+# Change the apiVersion from 'projectcalico.org/v1' to 'crd.projectcalico.org/v1' and save it in a new yaml file.
+# Apply the modified yaml file.
+/sbin/kubectl get globalbgpconfig --all-namespaces -o yaml > tpr-bgpconfig.yaml
+cat tpr-bgpconfig.yaml | sed '/apiVersion/s/projectcalico\.org\/v1/crd\.projectcalico\.org\/v1/g' > crd-bgpconfig.yaml
+/sbin/kubectl apply -f crd-bgpconfig.yaml
+
+# Delete the TPRs
```

Comment:
oh no, I'm going to add the command, but I wanted to push it so @caseydavenport can look at it before leaving :)